Gruntfile.js

```diff
@@ -13,7 +13,7 @@ module.exports = function(grunt) {
 
     eslint: {
       options: {
-        config: '.eslintrc'
+        config: '.eslintrc.json'
```

Comment:
Anyhow the version `2.0.0` of ESLint deprecated the extensionless configuration file, hence added it.